waku/factory/external_config.nim

```diff
@@ -312,8 +312,13 @@ type WakuNodeConf* = object
       name: "keep-alive"
     .}: bool
 
+    networkShards* {.
+      desc: "Number of shards in the network", defaultValue: 0, name: "network-shards"
+    .}: uint32
```

Comment:
Nitpick one :) Maybe a bit more explicit
```suggestion
    numShardsInNetwork* {.
      desc: "Number of shards in the network", defaultValue: 0, name: "num-shards-in-network"
    .}: uint32
```

Reply:
We can make it more explicit that it refers to the number, yes :) Committed the fix